extra_docs/relations/README.md

```diff
@@ -51,6 +51,25 @@ dom.dataset("posts")
 
 Notice that records should be inserted in appropriate order - the relation root must be insert root. 
 It's impossible to insert into comment dataset with post related to that comment (tree cannot grow upside down). 
+This example is not going to work:
```

Comment:
Andrey, it is not what I've meant. How I can add new comment to POST with ID =123 ?

Reply:
You are talking about attach/detach feature, it's not a part of this current pull request